inlineplz/linters/__init__.py

```diff
@@ -662,268 +657,289 @@ def vendored_path(path):
 }
 
 
-def run_command(command, log_on_fail=False, log_all=False, timeout=120):
-    print('Running: "{}"'.format(" ".join(command)))
-    shell = False
-    if os.name == "nt":
-        shell = True
-    popen_kwargs = {
-        "args": command,
-        "stdin": subprocess.PIPE,
-        "stdout": subprocess.PIPE,
-        "stderr": subprocess.PIPE,
-        "shell": shell,
-        "env": os.environ,
-        "universal_newlines": True,
-        "timeout": timeout,
-    }
-    if sys.version_info[0] >= 3 and sys.version_info[1] >= 6:
-        popen_kwargs["encoding"] = "utf-8"
-    try:
-        proc = subprocess.run(**popen_kwargs)
-    except subprocess.TimeoutExpired:
-        print("Timeout: {}".format(command))
-        return 0, ""
-
-    stdout, stderr = proc.stdout, proc.stderr
-    output = "{}\n{}".format(stdout, stderr).strip()
-    if output and ((log_on_fail and proc.returncode) or log_all):
-        print(output)
-        sys.stdout.flush()
-    return proc.returncode, output
-
+class LinterRunner:
```

Comment:
[inline-plz]: `prospector: pylint: Too many instance attributes (11/7) (too-many-instance-attributes)`

Reply:
Will fix this in another PR